CONTRIBUTING.md

```diff
@@ -5,118 +5,9 @@
 Follow the steps bellow to add a new feature, make an improvement or fix a bug.
 (Make sure to read the guides on each step as they contain important additional information)
 
```

Comment:
We should add a warning that the following process is fully tested only for internal users that have access to open a branch, and make use of our pipelines.Maybe reinforce we are open to issues and would like to improve this process to external users.

Reply:
Inclui nesse arquivo um "disclaimer" que o nosso processo de contribuição ta preparado somente pra contribuições internas no momento, e que pretendemos melhorar isso no futuro ;)
done in c5e361b